base/db/R/get.trait.data.R

```diff
@@ -56,7 +56,7 @@ get.trait.data <- function(pfts, modeltype, dbfiles, database, forceupdate,
     # `query_priors`, but haven't done so yet because that requires
     # prepared statements and therefore requires the Postgres driver. 
     all_priors_list <- lapply(format(pft_ids, scientific = FALSE), query.priors,
-                              con = dbcon, trstr = trait.names)
+                              con = dbcon, trstr = trait.names, update = settings$db$update)
```

Comment:
I'm not seeing where `settings` is defined inside `get.trait.data`, so I'm surprised this doesn't still throw a global variable warning. Am I missing a definition somewhere above?

Reply:
Hi @infotroph, I added the `update` variable in the `query.priors` function, and here is the call to that function that's why I passed `update` as an argument. I have done some changes now and removed `settings` from `get.trait.data`. Please review it once